src/WebJobs.Script/Eventing/Rpc/WorkerProcessReadyEvent.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Eventing
+{
+    public class WorkerProcessReadyEvent : ScriptEvent
```

Comment:
Is this replacing the WorkerReadyEvent?

Reply:
This event is specifically raised when restarting language worker process fails after 3 retries